core/src/main/scala/org/apache/spark/util/RpcUtils.scala

```diff
@@ -48,11 +48,13 @@ object RpcUtils {
 
   /** Returns the default Spark timeout to use for RPC ask operations. */
   def askTimeout(conf: SparkConf): FiniteDuration = {
-    conf.getTimeAsSeconds("spark.rpc.askTimeout", "30s") seconds
+    conf.getTimeAsSeconds("spark.rpc.askTimeout",
+      conf.get("spark.network.timeout", "30s")) seconds
```

Comment:
can we change it to the same timeout where we set for spark.network.timeout else where? I think we use a number higher than 30s.